### ISSUE TITLE
function-paren-newline reports wrong messages

### ISSUE THREAD

**Issue by fasttime** [eslint/eslint#12134]
**Tell us about your environment**

* **ESLint Version:** 6.2.0
* **Node Version:** 12.9.0
* **npm Version:** 6.10.2

**What parser (default, Babel-ESLint, etc.) are you using?**
default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
{
    "rules": {
        "function-paren-newline": "error"
    }
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
a(
 );
```

[Demo link](https://eslint.org/demo#eyJ0ZXh0IjoiYShcbiApO1xuIiwib3B0aW9ucyI6eyJwYXJzZXJPcHRpb25zIjp7ImVjbWFWZXJzaW9uIjo1LCJzb3VyY2VUeXBlIjoic2NyaXB0IiwiZWNtYUZlYXR1cmVzIjp7fX0sInJ1bGVzIjp7ImZ1bmN0aW9uLXBhcmVuLW5ld2xpbmUiOjJ9LCJlbnYiOnt9fX0=)

**What did you expect to happen?**
>  1:2   error  Unexpected newline after '('   function-paren-newline
>  2:2   error  Unexpected newline before ')'  function-paren-newline

**What actually happened? Please include the actual, raw output from ESLint.**

The parentheses are actually swapped.

>  1:2   error  Unexpected newline after ')'   function-paren-newline
>  2:2   error  Unexpected newline before '('  function-paren-newline

**Are you willing to submit a pull request to fix this bug?**
Sure.

**Comment by kaicataldo** [comment]
Verified using the demo link above. Thanks for reporting this!
